ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -1809,6 +525,7 @@
 /* Begin XCBuildConfiguration section */
 		00E356F61AD99517003FC87E /* Debug */ = {
 			isa = XCBuildConfiguration;
+			baseConfigurationReference = 047663CF32A867699B13342E /* Pods-ZulipMobileTests.debug.xcconfig */;
 			buildSettings = {
```

Comment:
There are some mentions of our newly-podified dependencies in several HEADER_SEARCH_PATHS lists below. Those can probably get deleted too.

(Can't highlight the actual lines, because GitHub.)

Found with `rg react-native ios/ZulipMobile.xcodeproj/`.

Reply:
Hmm, I've just found out that some of these can't be deleted without there being build failures; I'm investigating now.